test/acceptance/tests.py

```diff
@@ -910,6 +913,65 @@ def verify_grade_entries(self, expected_entries):
             self.assertEqual(expected_entry[1], self.grade_page.grade_entry(1, index))
 
 
+class MultipeOpenAssessmentMixin(FullWorkflowMixin):
+    """
+    A Multiple ORA assessment mixin with helper methods and constants for testing a full workflow
+    (training, self assessment, peer assessment, staff override).
+    """
+
+    def setup_vertical_index(self, vertical_index):
+        """
+        Set the vertical index on the page.
+        Each problem has vertical index assigned and has a `vert-{vertical_index}` top level class.
+        Set up vertical index on the page so as to move to a different problem.
+        """
+        self.submission_page.vertical_index = vertical_index
+        self.self_asmnt_page.vertical_index = vertical_index
+        self.peer_asmnt_page.vertical_index = vertical_index
+        self.student_training_page.vertical_index = vertical_index
+        self.staff_asmnt_page.vertical_index = vertical_index
+        self.grade_page.vertical_index = vertical_index
+        self.staff_area_page.vertical_index = vertical_index
+
+    def staff_assessment(self, peer_grades_me=True):
+        """ Do staff assessment workflow """
+
+        # Ensure grade is not present, since staff assessment has not been made
```

Comment:
Since this is using the base `FullWorkflowMixin` class, could you refactor this logic to be shared and not duplicated?

https://github.com/edx/edx-ora2/blob/mushtaq/tnl-4155-fix-multiple-ora-issue/test/acceptance/tests.py#L939-L966 and https://github.com/edx/edx-ora2/blob/mushtaq/tnl-4155-fix-multiple-ora-issue/test/acceptance/tests.py#L1147-L1174 seem to be identical.